packages/components/src/progress-bar/index.tsx

```diff
@@ -20,11 +20,11 @@ function UnforwardedProgressBar(
 	props: WordPressComponentProps< ProgressBarProps, 'progress', false >,
 	ref: ForwardedRef< HTMLProgressElement >
 ) {
-	const { className, value, ...progressProps } = props;
+	const { className, value, width, ...progressProps } = props;
```

Comment:
Yeah, I thought about that. But in the end, it does set the width, just that it will fit the container. I can rename it to `max-width` if it makes it more clear/explicit, np 👍🏻